iphone/Classes/UIModule.m

```diff
@@ -739,6 +745,16 @@ - (NSString *)CLIPBOARD_OPTION_EXPIRATION_DATE
 MAKE_SYSTEM_PROP(TABLE_VIEW_SEPARATOR_STYLE_NONE, UITableViewCellSeparatorStyleNone);
 MAKE_SYSTEM_PROP(TABLE_VIEW_SEPARATOR_STYLE_SINGLE_LINE, UITableViewCellSeparatorStyleSingleLine);
 
+#if defined(USE_TI_UISHORTCUT) || defined(USE_TI_UISHORTCUTITEM)
+- (TiUIShortcutProxy *)Shortcut
+{
+  if (shortcut == nil) {
+    shortcut = [[TiUIShortcutProxy alloc] _initWithPageContext:[self executionContext]];
```

Comment:
```suggestion
    shortcut = [[TiUIShortcutProxy alloc] init];
```
And restoring the implementation in TiUIShortcutProxy should work. No need to change ObjcProxy style to old one.